pages.it/linux/i3.md

```diff
@@ -0,0 +1,37 @@
+# i3
+
+> Window Manager Dinamico.
+> Tasto Windows=Super=Mod4, Alt=Mod1
```

Comment:
This is a common Key mapping stuff for Linux users, but it's OK if you think this is needed, we can ask for opinions to other collaborators.

Reply:
The other translated pages don't have it and it's just a default and can be easily changed.
The pages has a few issues in general. I'll probably fix this in a seperate PR.

Reply:
If we're keeping this line, we should add a dot.
```suggestion
> Tasto Windows=Super=Mod4, Alt=Mod1.
```

Reply:
I'm more concerned that someone doesn't know super=windows, or if it was changed to Mod1, it would rendering the rest of the page not useful.

Reply:
Could someone translate this back to English for me? I don't understand it (unfortunately I don't understand Italian), and it doesn't look like it appears in the English version of the page.

Reply:
@sbrl The translation to English could be:
```markdown
> Key Windows=Super=Mod4, Alt=Mod1
```
It's like the alias `Mod4` or `Super` is the `Windows` key in the keyboard.

Reply:
Right, thanks @navarroaxel. I'm unsure that this is related specifically to this command - rather this is a difference across platforms. I'm not sure whether this works being on a tldr page like this, because then we have the question of which pages to include it on. It's perhaps better served as something the user looks up - which they'll have to do only once before they then remember it I would guess.

Reply:
On first launch, the i3 prompts the user to select Mod1 or Mod4 as the modifier. My concerns are twofold: (1) that the new user may not know which key `Super` is, (2) it's just as easy for someone to select (even accidentally) Alt in place of Super; in either of these cases it renders this tldr page useless. I understand that conformity across languages takes precedence, but just some food for thought.

Reply:
Right, so this is specific to `i3`?